### ISSUE TITLE
Pension List+Loop:

### ISSUE THREAD

**Issue by kristinoletmuskat** [department-of-veterans-affairs/va.gov-team#105387]
## Brief description
As part of our list and loop enhancements, we submitted an [ADE ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/104338) for a11y review. 

## Problem - Missing VADS component
**Page**: Add a dependent child, [Address information](https://staging.va.gov/pension/apply-for-veteran-pension-form-21p-527ez/household/dependents/0/address?edit=true) page
**Device**: MacOS
**Expected severity level:** 4
**Expected experience standard**: User encounters consistent, clear design components and patterns
**Description**: 
The “How much do you contribute per month to your child’s support?” **text input isn’t a VADS component**. It may not behave in the same way as the other text inputs as a result. In this screenshot, VADS component is highlighted: 

![Image](https://github.com/user-attachments/assets/cf8cc367-4459-4214-9181-760522f1cdfd)

**How to recreate**: 
Go to the Add a dependent child flow, Does this child live with you? Page. Select “No” to get to the address page. Inspect “How much do you contribute per month to your child’s support?”

**Recommendation**: Replace this with the [va-text-input with currency](https://design.va.gov/storybook/?path=/docs/uswds-va-text-input--docs#with-currency) component.
